ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -97,7 +117,8 @@ class LocalAssemblerData : public LocalAssemblerDataInterface<GlobalMatrix, Glob
         for (std::size_t ip(0); ip < n_integration_points; ip++) {
             auto const& sm = _shape_matrices[ip];
             auto const& wp = integration_method.getWeightedPoint(ip);
-            _localA->noalias() += sm.dNdx.transpose() * _hydraulic_conductivity *
+            _localA->noalias() += sm.dNdx.transpose() *
+                _hydraulic_conductivity().asDiagonal() *
```

Comment:
I suggestion to replace ``_hydraulic_conductivity().asDiagonal()`` with a template parameter, which then can be a scalar or tensor variable for isotropic, anisotropic or transversely isotropic/anisotropic. Under such consideration, , the computation time is saved for the case of scalar parameter.